src/mqt/predictor/rl/helper.py

```diff
@@ -68,9 +66,8 @@
 
 
 if TYPE_CHECKING or sys.version_info >= (3, 10, 0):
-    from importlib import metadata, resources
+    from importlib import resources
```

Comment:
The type checking block here is no longer necessary. We only run on Python 3.10+

Reply:
Removed it.